### ISSUE TITLE
php-ampqlib should explicitly list ext-mbstring as a dependency in composer

### ISSUE THREAD

**Issue by FractalizeR** [php-amqplib/php-amqplib#229]
Latest 2.4.1 release still doesn't have mbstring extension as a requirement in a composer.json from HEAD.

**Comment by FractalizeR** [comment]
Sure. After 14th, when I am back from vacations. Left notebook at home to
fully clean my mind :)

Best regards,
Vladislav "FractalizeR" Rastrusny

... sent from mobile
05 Янв 2015 г. 23:31 пользователь "Alvaro Videla" <notifications@github.com>
написал:

**Comment by videlalvaro** [comment]
@FractalizeR no worries. Enjoy your holidays!

**Comment by FractalizeR** [comment]
I'm back ;) But I am unsure about how to do this change. Required commit is already in the master branch: ce78b644f1ccf91b64af5ddc8cddd8c44317d14f. It just was performed later, than 2.4.1 was released.

In this situation I suppose there should be 2.4.2, incorporating just this commit and nothing else from master. I've branched v2.4.1 and cherry-picked necessary commit there. But I cannot currently send any PR because such PR shouldn't be merged into master, I suppose. I think a branch with similiar name can be created in the master repo, PR could be then sent there and the latest commit in that new branch could be tagged against v2.4.2 and left as is.
